examples/plot_custom_precipitation_range.py

```diff
@@ -0,0 +1,152 @@
+#!/bin/env python
+"""
+Plot precipitation using custom colormap
+=============
+
+This tutorial shows how to plot data using a custom colormap with a specific
+range of precipitation values.
+
+"""
+
+import os
+from datetime import datetime
+import matplotlib.pyplot as plt
+
+import pysteps
+from pysteps import io, rcparams
+from pysteps.utils import conversion
+from pysteps.visualization import plot_precip_field
+from pysteps.datasets import download_pysteps_data, create_default_pystepsrc
+
+
+###############################################################################
+# Download the data if it is not available
```

Comment:
Sounds good, I'll keep  it as is. My thought process was that the data erequires a manual download, so providing a way for users to quickly see how the plotting works in case they do not have it loaded.
Is the data included in the installation of the library via pip?

Reply:
no, the data are not part of the actual package, but we provides some tools to get them (as the functions you used), see https://pysteps.readthedocs.io/en/stable/user_guide/example_data.html